docs/fallbacks.md

```diff
@@ -1,28 +1,66 @@
 # Fallbacks
 
-### Circuit Breaker
+## Circuit Breaker
 
 A simple implementation of [Circuit Breaker](http://martinfowler.com/bliki/CircuitBreaker.html) pattern.
 
+### MaxFailuresRule
+Rules to open circuit based on maximum number of failures
+
+#### class MaxFailuresRule(max_failures, failure_cache_key)
+
 #### Arguments
 
-`cache`
+`max_failures`
 
-Django cache object.
+Maximum number of errors.
 
 `failure_cache_key`
 
 Cache key where the number of errors is incremented.
 
+### PercentageFailuresRule
+Rules to open circuit based on a percentage of failures.
+
+#### class PercentageFailuresRule(max_failures, failure_cache_key, max_accepted_failures, request_cache_key)
+
+#### Arguments
+
 `max_failures`
 
-Maximum number of errors.
+Maximum percentage of errors.
+
+`failure_cache_key`
+
+Cache key where the number of errors is incremented.
+
+`max_accepted_failures`
+
+Maximum number of errors accepted to not open circuit breaker.
+
+`request_cache_key`
+
+Cache key where the number of requests is incremented.
+
+### CircuitBreaker
+
+#### class CircuitBreaker(rule, cache, failure_exception, failure_timeout=None, circuit_timeout=None, catch_exceptions=None)
+
+#### Arguments
+
+`rule`
 
-`max_failure_exception`
+Instance of class Rule.
```

Comment:
If you move the rules to the bottom, you can add a link to the "Rule" title here.